magento/magento2
# Fix typo mistake in mediaGallery spelling

Fix typo mistake in mediaGallery spelling in Magento\CatalogImportExport\Model\Export\Product.php file

### Description (*)
There is file name Product.php at path Magento\CatalogImportExport\Model\Export\Product has typo mistake in 6 places.
I also checked that this key is not used anywhere other than this file. 

### Fixed Issues (if relevant)


### Manual testing scenarios (*)
1. open file Magento\CatalogImportExport\Model\Expor\Product.php
2. See there is mediaGalery written  at 6 places instead of mediaGallery

### Questions or comments

### Contribution checklist (*)
